.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,41 @@
+name: aws-deploy
+
+on: push
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.NEXTFLOW_PROD_CI_ACCESS_KEY }}
```

Comment:
you'll need to setup a service user for this, you can use this template to setup a user/role combination for this.  https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/IAM/service-account.yaml

hmm, we prefer the CI to assume a role to do it's thing and that doesn't look like something you can do with the sceptre GH action.  i think we have 3 options:
1. create ~/.aws/config and ~/.aws/credentials files and set a AWS_PROFILE env var.
2. use the GH assume role action, not sure if that's gonna work with sceptre GH action though.
3. add the feature to sceptre GH action to assume a role.

Reply:
@zaro0508: Do you think specifying the [role ARN](https://sceptre.cloudreach.com/2.4.0/docs/stack_config.html#role-arn) in the sceptre config would help circumvent the lack of support for roles in the sceptre action?

Reply:
ahh yes, good thinking @BrunoGrandePhD, i think that should work.  the role arn doesn't need to be a secret

Reply:
@zaro0508: After taking a look at the `aws-actions/configure-aws-credentials@v1` [action](https://github.com/aws-actions/configure-aws-credentials/blob/master/index.js#L114-L121) and the sceptre [codebase](https://github.com/Sceptre/sceptre/blob/v2.4.0/sceptre/connection_manager.py#L139-L141), I think the `aws-actions/configure-aws-credentials@v1` action will work. 

I'm holding off on using the `role_arn` config value because I prefer keeping everything related to the CI within that YAML file, if possible.